main.py

```diff
@@ -4,45 +4,13 @@
 parser = argparse.ArgumentParser()
```

Comment:
I would probably add a 'script-wide' comment at the start of this file to explain it's purpose.